src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -400,12 +401,9 @@ def delete(path: Path) -> None:
         spath.rmdir(recursive=True)  # remove dir and all contains
 
 
-def symlink_or_copy(
-    src: Path, dst: Path, deploy_root: Path, makedirs=True, overwrite=True
-) -> None:
+def symlink_or_copy(src: Path, dst: Path, deploy_root: Path, makedirs=True) -> None:
```

Comment:
why keep makedirs though? We don't ever set it to false, and it muddies the contract (+ requires extra testing). How about we just delete it and remove the tests that covered that unnecessary behaviour?